client/src/components/tables/components/TachiTable.tsx

```diff
@@ -170,7 +170,9 @@ export default function TachiTable<D>({
 					<tbody>
 						<NoDataWrapper>
 							{window.map((e, i) => (
-								<React.Fragment key={i}>{e && rowFunction(e)}</React.Fragment>
+								<React.Fragment key={i + 10 * (page - 1)}>
```

Comment:
Hardcoding to `10` would break for different page sizes. You should probably use `pageLen` instead.